test/pylib/scylla_cluster.py

```diff
@@ -298,7 +298,17 @@ def __init__(self, mode: str, exe: str, vardir: str | pathlib.Path,
         self.control_cluster: Optional[Cluster] = None
         self.control_connection: Optional[Session] = None
         shortname = f"scylla-{f'{xdist_worker_id}-' if xdist_worker_id else ''}{self.server_id}"
-        self.workdir = self.vardir / shortname
+
+        workdir = self.vardir / shortname
```

Comment:
Please open issue and send as independent PR (and consider backport I guess to all live versions)

Reply:
We do not need a backport. Existing tests do not need to overwrite the workdir except those that we introduce. Hence I put it as part of this PR and not a separate one.